setup.py

```diff
@@ -85,7 +92,7 @@
         "14.0": "Visual Studio 14 2015",
         "14.1": "Visual Studio 15 2017",
         "14.2": "Visual Studio 16 2019",
-        "14.3": "Visual Studio 17 2022",
+        "14.3": "Visual Studio 16 2019",  # ?"14.3": "Visual Studio 17 2022",  # ?
```

Comment:
`14.3` is `Visual Studio 17 2022`, if we need to change this we should just make sure we can override via env var

Reply:
Not sure why we we're hitting this (gagan made the change first, I confirmed I needed it )
I can try again with the windows-2019 build

Reply:
got it, so good to resolve?

Reply:
So this works in the build but breaks local builds since it will try 2022 again.

Reply:
That just means your local python was built with `14.3`, but for whatever reason we dont build properly on `14.3`. So you'd `set CSP_GENERATOR="Visual Studio 16 2019"` before building

Reply:
we can make a separate ticket for "support Visual Studio 17 2022" and figure out why thats not working later on

Reply:
https://github.com/Point72/csp/issues/240